docs/source/eventlog.rst

```diff
@@ -0,0 +1,61 @@
+Eventlogging and Telemetry
```

Comment:
Let's add a security note about not relying on this for security or auditing purposes, unless the user can categorically not mess with the python environment this is running in? Otherwise you can just change the code to emit whatever you want. Likewise, user code can just hit the eventlogging endpoint to add any spurious events it wants.

Almost everyone I talk to wants to use this for security, and this caveat is very important.